src/mappings/omim.sssom.tsv

```diff
@@ -137,7 +137,7 @@ MONDO:0004914		skos:exactMatch	OMIM:116870	celiac artery stenosis from compressi
 MONDO:0004933		skos:exactMatch	omim.ps:241550	Hypoplastic left heart syndrome	semapv:UnspecifiedMatching
 MONDO:0004948		skos:exactMatch	OMIM:151400	leukemia, chronic lymphocytic	semapv:UnspecifiedMatching
 MONDO:0004951		skos:exactMatch	OMIM:609423	human immunodeficiency virus type 1, susceptibility to	semapv:UnspecifiedMatching
-MONDO:0004980		skos:exactMatch	omim.ps:603165	Dermatitis, atopic	semapv:UnspecifiedMatching
+MONDO:0004980		skos:exactMatch	OMIM:603165	dermatitis, atopic	semapv:UnspecifiedMatching
```

Comment:
@joeflack4 can you confirm that the change from `omim.ps:603165` to `OMIM:603165` was intended and not a bug due to updates in the OMIM processing code? Keep in mind that OMIM uses the same integer identifier, e.g. for this line 603165, for both omim entry pages (https://omim.org/entry/603165) and for omim phenotypic series pages (https://omim.org/phenotypicSeries/PS603165).

It does look like this has been happening in [past release data builds](https://github.com/monarch-initiative/mondo-ingest/commit/52ec97cfe3d64c5aa49f49393fe2735c2a45ca78#diff-18877ef566bfe924f453aceef30378143e251ad4b74d410cf8fccbe63262103bR4344) on `main`.


@sabrinatoro maybe you have comments on this content in the "omim.sssom.tsv" file?

Reply:
It's possible that @sabrinatoro may know more. The source of this (and other similar changes of matches from OMIMPS to OMIM terms, and possibly vice versa) is due to changes in `mondo.sssom.tsv`. It could be that this is the result of curation efforts, or something else. I'm not sure all of the inputs that go into `mondo.sssom.tsv` changing.

I reached this conclusion by examining my local mapping files and outputs in both `omim` and `mondo-ingest` repositories.

I observed that the OMIM build on GitHub is using the latest SSSOM files. However my local environment still had some old ones. Those old ones still had this `omim.ps:603165` mapping, which was included in my local `omim.owl` (due to using these local files as inputs). But observing `mondo-ingest` and the `omim` GH release, which use up-to-date files, the mapping is for `OMIM:603165` instead.